proxy/proxy-html/spacewalk-proxy-html.spec

```diff
@@ -49,18 +56,17 @@ displays, if you navigate to it using your browser.
 
 %install
 install -m 755 -d $RPM_BUILD_ROOT%{htmldir}
-install -m 755 -d $RPM_BUILD_ROOT%{htmldir}/_rhn_proxy
-install -m 644 _rhn_proxy/* $RPM_BUILD_ROOT%{htmldir}/_rhn_proxy/
+install -m 644 %{proxy_dir_name}/* $RPM_BUILD_ROOT%{htmldir}/
 
 %files
 %defattr(-,root,root)
-%dir %{htmldir}/_rhn_proxy
-%config %{htmldir}/_rhn_proxy/index.html
-%{htmldir}/_rhn_proxy/*.ico
-%{htmldir}/_rhn_proxy/*.png
+%dir %{htmldir}
+%config %{htmldir}/index.html
+%{htmldir}/sources/img/*.ico
+%{htmldir}/sources/img/*.png
 %doc LICENSE
 %if 0%{?suse_version}
-%dir %dir %{htmldir}/_rhn_proxy
+%dir %dir %{htmldir}
```

Comment:
You are already included `%{htmldir} at line 64 for all OS.

Looks to me that this whole if can go away?

Reply:
Good point @juliogonzalez , it was the case when there was the subfolder `_rhn_proxy` before this PR, but now it is removed and the whole `if` condition can be dropped. Thanks